bc/channel/models.py

```diff
@@ -44,6 +67,9 @@ class Channel(AbstractDateTimeModel):
     enabled = models.BooleanField(
         help_text="Disabled by default; must enable manually", default=False
     )
+    alias = models.ForeignKey(
```

Comment:
Per my last comment, maybe this should be:

```suggestion
    group = models.ForeignKey(
```

?

Reply:
Yes. I will update the name of this field.